tasks/README.md

```diff
@@ -84,7 +108,9 @@ The preferred style of the keys should be `snake_case`.
   "implementations" : [
     {"name": "foo.sh", "requirements": ["shell"]},
     {"name": "foo.ps1", "requirements": ["powershell"]}
-  ]
+    {"files": "my_util/files/util/win_tools.ps1", "requirements": ["powershell"]}
```

Comment:
`name` will always be required, it's the file that we run as a task. `files` is an optional property that specifies additional files to copy over.

Note that `requirements` is also optional, lack of it means we'll use it anywhere.